widgets/pagination.js

```diff
@@ -1,6 +1,8 @@
 var React = require('react');
 
 var utils = require('../lib/utils.js');
+var autoHide = require('../decorators/autoHide');
+var Pagination = autoHide(require('../components/Pagination/Pagination.js'));
```

Comment:
I would change the name of Pagination to make more explicit the composed aspect of this new Pagination component. Same goes for any component using this technique in the widgets.